src/content/faq/General.md

```diff
@@ -9,70 +9,73 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
-<!-- TODO -->
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance and stability by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI clients](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
```

Comment:
@carolstran Do you mean `multiple UI components`?

Reply:
Yes! Good catch, I'll update.